projects/slap_manipulation/src/slap_manipulation/policy/interaction_prediction_module.py

```diff
@@ -457,10 +457,10 @@ def predict(
         lang: language annotation which is encoded
         proprio: proprioception, an np.ndarray (gripper-action, gripper-width, time)
         """
-        t_rgb = torch.FloatTensor(feat[0]).to(self.device)
```

Comment:
this will still work with the RLBench and Franka data right?

Reply:
It's a new function which doesn't disrupt how RLBench + Franka use IPM/APM. I am planning on porting over all code to use this in subsequent PRs.